jenkins_testing/Dockerfile.centos_7.client_runner

```diff
@@ -13,6 +13,11 @@ COPY irods_rest_pkg/irods*rpm /irods_packages_in/
 # _-_-_ TODO - Calculate or parameterize which build of pistache & other externals to install
 
 RUN rm -f $(find /irods_packages_in -type f -size 0)
+RUN . /tmp/PKGFILE_install_dependencies && \
+    PREREQ=$( get_yum_prerequisites_list /irods_packages_in/irods-externals-*.rpm ) && { \
+        [ -z "$PREREQ" ] || yum install -y $PREREQ  ;\
```

Comment:
maybe quotes around second $PREREQ ?

Reply:
Well ... I need expansion into several arguments (each a package name) in this case, so better not.
Arguablly $PREREQ should be an array , but we're safe here because package names *never* themselves include spaces.

Reply:
fair enough, wanted to make sure you thought through it.